ProjetINFRA/src/test/java/fr/m2dl/infra/OrchestratorTest.java

```diff
@@ -0,0 +1,27 @@
+package fr.m2dl.infra;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class OrchestratorTest {
+
+	@Test
+	public void createOrchestratorTest() {
+		Orchestrator orchestrator = new Orchestrator();
```

Comment:
This maybe can be done in a @Setup and the Orchestrator will be use in every test. That reduce the amount of lines of code in each tests.